requirements.txt

```diff
@@ -37,7 +37,7 @@ python-dateutil==2.8.2
 pytz==2022.1
 redis==4.2.2
 requests==2.27.1
-sqlalchemy==1.4.34  # Required by Celery broker transport
+sqlalchemy==1.4.35  # Required by Celery broker transport
```

Comment:
I have no idea why this is marked as a change, it is the same version as in `dev`.